.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
     "singleQuote": true,
     "tabWidth": 4,
-    "trailingComma": "all"
+    "trailingComma": "none"
```

Comment:
`none` is the default so we can remove the option completely. https://prettier.io/docs/en/options.html#trailing-commas

Reply:
We leave it in here intentionally. Certain Aura JS constellations may have issues with that setting, so it's more a "leave everything as we recommend in here".